app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The comment should also belong to an `Author` not a user, kindly fix this.
e.g
`belongs_to :author, class_name: 'User'`